src/sage/schemes/elliptic_curves/ell_field.py

```diff
@@ -1363,6 +1363,10 @@ def isogenies_prime_degree(self, l=None, max_l=31):
         - ``max_l`` -- (default: 31) a bound on the primes to be tested.
           This is only used if ``l`` is None.
 
+        - ``cyclic_after`` -- (default: None) an isogeny with codomain equal to
```

Comment:
```suggestion
        - ``cyclic_after`` -- (default: ``None``) an isogeny with codomain equal to
```
I am not sure if `cyclic_after` is a good name, but I can't think of a better one.